doc/nrf/releases/release-notes-latest.rst

```diff
@@ -93,4 +93,8 @@ The following list summarizes the most important changes inherited from upstream
 
 The following list contains |NCS| specific additions:
 
-*
+* Added support for the |NCS|'s :ref:`partition_manager`, which can be used for flash partitioning.
+* Added the following network socket and address extensions to the :ref:`zephyr:bsd_sockets_interface` interface to support the functionality provided by the :ref:`nrfxlib:bsdlib`:
+  AF_LTE, NPROTO_AT, NPROTO_PDN, NPROTO_DFU, SOCK_MGMT, SO_RCVTIMEO, SO_BINDTODEVICE, SOL_PDN, SOL_DFU, SO_PDN_CONTEXT_ID, SO_PDN_STATE, SOL_DFU, SO_DFU_ERROR, TLS_SESSION_CACHE, SO_SNDTIMEO, MSG_TRUNC, SO_SILENCE_ALL, SO_IP_ECHO_REPLY, SO_IPV6_ECHO_REPLY
+* Added support for enabling TLS caching when using the :ref:`zephyr:mqtt_socket_interface` library. See also `Enabling and testing TLS in mqtt_simple`_
```

Comment:
Please put the second sentence on a new line.

That link explains how to enable TLS, but is this what we mean with "TLS caching"? Then we should maybe word it differently, because it sounds like this is something different than what is described in the blog post. Or is there a new Kconfig option (or some other configuration) for TLS caching?

Reply:
> That link explains how to enable TLS, but is this what we mean with "TLS caching"?

Actually not. I just browsed the code.
I can only see that the option is now possible, but I don't find any place where we actual use this feature.
Maybe we should leave it out until we actual have a sample that shows how this can be used.
@jtguggedal any input on this ?

Will remove link again, as it only describes TLS itself and not the caching option.

Reply:
TLS caching option has been added to the downstream version of the MQTT library (no support in native network stack, so can't be added upstream yet), so its use is shown there:
https://github.com/nrfconnect/sdk-zephyr/blob/26216d1f7e09e9c43dbcfaf0a8565bb0f69d0130/subsys/net/lib/mqtt/mqtt_transport_socket_tls.c#L81

I have a new library in the making that uses the socket option, and it will hopefully feature in NCS v1.4.0.
I can also add TLS caching to the MQTT sample in time for NCS 1.4.0 to show how it can be used.

Reply:
> TLS caching option has been added to the downstream version of the MQTT library (no support in native network stack, so can't be added upstream yet), so its use is shown there:

Well, that is also the code I found, but that just checks if the config enables it, and then sets the sock option accordingly.

What I referred to was somewhere that a sample was doing, for examlple
```
tls_config->session_cache = TLS_SESSION_CACHE_ENABLED;
```
similar to:
https://github.com/nrfconnect/sdk-nrf/blob/master/samples/nrf9160/aws_fota/src/main.c#L431-L438

and I don't find such example anywhere.
Therefore the question, if we should leave it out.

> I have a new library in the making that uses the socket option, and it will hopefully feature in NCS v1.4.0.
> I can also add TLS caching to the MQTT sample in time for NCS 1.4.0 to show how it can be used.

When that is available, then we should probably update release notes, and thus leave it out for now?

Reply:
If it's practice to leave things out if it's not shown how to use it in a sample, I'm fine with that.
We will highlight TLS caching more in v1.4.0 and make sure it goes into the release notes then.

Reply:
There's no common practice since this is the first time we're adding these release notes. ;)
But that option is actually documented, so how about:
```suggestion
* Added support for enabling TLS caching when using the :ref:`zephyr:mqtt_socket_interface` library.
  See :c:macro:`TLS_SESSION_CACHE`.
```
This will then link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/reference/networking/sockets.html?highlight=tls_session_cache#c.TLS_SESSION_CACHE

Reply:
Done